persistence-modules/java-mongodb/src/main/java/com/baeldung/MongodbExample.java

```diff
@@ -0,0 +1,101 @@
+package com.baeldung;
+
+import static com.mongodb.client.model.Filters.eq;
+import static com.mongodb.client.model.Filters.gt;
+import static com.mongodb.client.model.Filters.lt;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.bson.Document;
+import org.bson.conversions.Bson;
+import org.bson.types.ObjectId;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoCursor;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.model.Projections;
+import com.mongodb.client.model.Sorts;
+import com.mongodb.client.model.Updates;
+
+public class MongodbExample {
```

Comment:
This class doesn't seem to be used in any tests? All code included in the PR has to have unit tests - can we remove this and rely on the MongodbCrudOperationsLiveTest?

Reply:
Absolutely, resolving now